src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/UpdateOrgContactInformationIntegrationTest.java

```diff
@@ -0,0 +1,113 @@
+            "Field error in object 'contactInformationCreationRequest' on field 'addressLine1'");
+    }
+
+    @Test
+    void update_contact_information_multiple_requests_existing_for_org_should_return_200() {
+
+        ContactInformationCreationRequest contactInformationCreationRequest =
+            createContactInformationRequest().build();
+
+        java.util.Map<String, Object> responseForOrganisationCreation = professionalReferenceDataClient
+            .createOrganisation(organisationRequestWithMultipleAddressAllFields().build());
+
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+            .updateOrgContactInformation(contactInformationCreationRequest,
+                hmctsAdmin,(String)responseForOrganisationCreation.get(ORG_IDENTIFIER),true);
+
+        assertThat(updateResponse).containsEntry("http_status", 200);
+    }
+
+    @Test
+    void update_contact_information_single_requests_existing_for_org_should_return_200() {
+
+        ContactInformationCreationRequest contactInformationCreationRequest =
+            createContactInformationRequest().build();
+
+
+        java.util.Map<String, Object> responseForOrganisationCreation = professionalReferenceDataClient
+            .createOrganisation(organisationRequestWithAllFieldsAreUpdated().build());
+
+
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+            .updateOrgContactInformation(contactInformationCreationRequest,
+                hmctsAdmin,(String)responseForOrganisationCreation.get(ORG_IDENTIFIER),true);
+
+        assertThat(updateResponse).containsEntry("http_status", 200);
+    }
+
+    @Test
+    void update_contact_information_for_non_existing_organisation_should_return_400() {
+        ContactInformationCreationRequest contactInformationCreationRequest =
+            createContactInformationRequest().build();
+
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+            .updateOrgContactInformation(contactInformationCreationRequest,  hmctsAdmin,
+                "ABCDEF7",true);
+
+        assertThat(updateResponse).containsEntry("http_status", "404");
+        assertThat(updateResponse.get("response_body").toString())
+            .contains("errorMessage\":\"4 : Resource not found\",\"errorDescription\":\"Organisation does not exist");
+
+    }
+
+    @Test
+    void update_contact_information_invalid_request_should_return_400() {
+
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+            .updateOrgContactInformation(new ContactInformationCreationRequest(null,null,
+                null,null,null,null,null,
+                null,null), hmctsAdmin,createOrganisationRequest(),true);
+
+        assertThat(updateResponse).containsEntry("http_status", "400");
+        assertThat(updateResponse.get("response_body").toString())
+            .contains("validation on an argument failed");
+
+    }
+
+}
```

Comment:
add test case to update address details without dx address and assert updated address details